progress/progress.go

```diff
@@ -237,7 +241,11 @@ func (b *Bar) spin(ch <-chan time.Time) {
 		// Writes the spinner frame at the beginning of the previous line, moving
 		// the cursor back to the beginning of the current line for any errors or
 		// informative messages.
-		fmt.Fprintf(b.out, "\r%v%v%v\r", up, spinner[idx], down)
+		if runtime.GOOS == "windows" {
+			fmt.Fprintf(b.out, "\r%v\r", spinner[idx])
```

Comment:
How does this end up looking on Windows? I don't even have a VM at the moment to try it out.

Reply:
Hi @lance, it looks like this:

https://user-images.githubusercontent.com/26138643/131407735-1c1476c1-2e04-490d-83b6-4196bea36975.mp4

Reply:
Note spinner is rendered on next line of the message. Previous code uses control chars to move it up (before rendering the spinner) and then move it down, but they don't work on Windows and output gets like:
```
←[1A←[K   Building function image
←[1A/←[K
```